apps/arm/ArmConfig.h

```diff
@@ -1,166 +1,80 @@
-        .zeroOffset_Degrees = 0.0
-    };
+static Encoder::t_encoderConfig shoulderEncoderConfig = {
+    .pin_PWM = ENC_PWM_TRNTBL, .degreesPerUnit = 360, .zeroOffset_Degrees = 0.0};
 
-    static Encoder::t_encoderConfig elbowEncoderConfig = {
-        .pin_PWM = ENC_PWM_TRNTBL,
-        .degreesPerUnit = 360,
-        .zeroOffset_Degrees = 0.0
-    };
+static Encoder::t_encoderConfig elbowEncoderConfig = {
+    .pin_PWM = ENC_PWM_TRNTBL, .degreesPerUnit = 360, .zeroOffset_Degrees = 0.0};
 
-    static Encoder::t_encoderConfig wristLeftEncoderConfig = {
-        .pin_ChannelA = ENC_QUAD_WRST_LHS_A,
-        .pin_ChannelB = ENC_QUAD_WRST_LHS_A,
+static Encoder::t_encoderConfig wristLeftEncoderConfig = {.pin_ChannelA = ENC_QUAD_WRST_LHS_A,
+                                                          .pin_ChannelB = ENC_QUAD_WRST_LHS_A,
 
-        .degreesPerUnit = 1.0,
-        .zeroOffset_Degrees = 0.0
-    };
+                                                          .degreesPerUnit = 1.0,
+                                                          .zeroOffset_Degrees = 0.0};
 
-    static Encoder::t_encoderConfig wristRightEncoderConfig = {
-        .pin_ChannelA = ENC_QUAD_WRST_RHS_A,
-        .pin_ChannelB = ENC_QUAD_WRST_RHS_B,
+static Encoder::t_encoderConfig wristRightEncoderConfig = {.pin_ChannelA = ENC_QUAD_WRST_RHS_A,
+                                                           .pin_ChannelB = ENC_QUAD_WRST_RHS_B,
 
-        .degreesPerUnit = 1.0,
-        .zeroOffset_Degrees = 0.0
-    };
+                                                           .degreesPerUnit = 1.0,
+                                                           .zeroOffset_Degrees = 0.0};
 
-    static Encoder::t_encoderConfig clawEncoderConfig = {
-        .pin_ChannelA = ENC_QUAD_CLAW_A,
-        .pin_ChannelB = ENC_QUAD_CLAW_B,
+static Encoder::t_encoderConfig clawEncoderConfig = {.pin_ChannelA = ENC_QUAD_CLAW_A,
+                                                     .pin_ChannelB = ENC_QUAD_CLAW_B,
 
-        .degreesPerUnit = 1.0,
-        .zeroOffset_Degrees = 0.0
-    };
+                                                     .degreesPerUnit = 1.0,
+                                                     .zeroOffset_Degrees = 0.0};
 
-    /*** ROTARY ACTUATOR CONFIG ***/
+/*** ROTARY ACTUATOR CONFIG ***/
 
-    static ActuatorController::t_actuatorConfig turnTableActuatorConfig = {
-        .velocityPID = {
-            .P = 1.0, 
+static ActuatorController::t_actuatorConfig turnTableActuatorConfig = {
+    .velocityPID =
+        {
+            .P = 1.0,
             .I = 0.0,
-            .D = 0.0, 
+            .D = 0.0,
             .bias = 0.0,
             .deadZoneError = 0.1,
-        }, 
+        },
 
-        .positionPID = {
-            .P = 1.0, 
-            .I = 0.0,
-            .D = 0.0,
-            .bias = 0.0,
-            .deadZoneError = 0.1
-        }
-    };
+    .positionPID = {.P = 1.0, .I = 0.0, .D = 0.0, .bias = 0.0, .deadZoneError = 0.1}};
```

Comment:
Do u know what flag makes this stacked like above? Why is it different?

Reply:
weird bug... I'll fix this one manually